sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1780,8 +1781,9 @@ def _flush_batch(self, destination):
     return [
         pvalue.TaggedOutput(
             BigQueryWriteFn.FAILED_ROWS,
-            GlobalWindows.windowed_value((destination, row)))
-        for row in failed_rows
+            GlobalWindows.windowed_value((destination, row, row_errors)))
```

Comment:
I wonder if there's a way to do this in a backwards compatible way?

if not, could you add a mention of this change in `CHANGES.md`, so that users will be aware of the difference in API?

Reply:
oh I see that you already updated it. Thanks. Makes sense!